src/Illuminate/Support/functions.php

```diff
@@ -28,6 +28,34 @@ function defer(?callable $callback = null, ?string $name = null, bool $always =
     }
 }
 
+if (! function_exists('Illuminate\Support\enum_value')) {
+    /**
+     * Return a scalar value for the given value that might be an enum.
+     *
+     * @internal
+     *
+     * @template TValue
+     * @template TDefault
+     *
+     * @param  TValue  $value
+     * @param  TDefault|callable(TValue): TDefault  $default
+     * @return ($value is empty ? TDefault : mixed)
+     */
+    function enum_value($value, $default = null)
+    {
+        if (is_string($value) && empty($value)) {
```

Comment:
Are you sure it should be && here and not ||?

Reply:
this is an outdated code